src/applications/find-forms/widgets/createInvalidPdfAlert.js

```diff
@@ -39,26 +49,41 @@ export async function onDownloadLinkClick(event) {
   // Default to true in case we encounter an error
   // determining validity through the API.
   let formPdfIsValid = true;
+  let formPdfUrlIsValid = true;
   let form = null;
 
   try {
     const forms = await fetchFormsApi(formNumber);
     form = forms.results.find(f => f.id === formNumber);
     formPdfIsValid = form?.attributes.validPdf;
+    if (formPdfIsValid) {
+      // URLS can be entered invalid, 400 is returned, this checks to make sure href is valid
+      const response = await fetch(downloadUrl);
+      if (!response.ok) formPdfUrlIsValid = false;
```

Comment:
I don't think we can do this because cross-domain PDFs like those hosted on vba.va.gov will produce a CORS error, unless I am missing something?

Reply:
@ncksllvn You where right. I added some additional layer of security.